python/ray/experimental/channel/cpu_communicator.py

```diff
@@ -69,20 +71,26 @@ async def wait_collective(self, op_id: int, data: "torch.Tensor", op: ReduceOp):
     def _apply_op(self, op: ReduceOp, tensors: List["torch.Tensor"]) -> "torch.Tensor":
         """Apply the specified reduction operation across a list of tensors."""
 
+        SUM = 0
```

Comment:
Do you need to manually specifcy the five values? Why not just use `ReduceOp.*`? It's less error prone.